Gruntfile.js

```diff
@@ -84,6 +84,7 @@ module.exports = (grunt) => {
           'bower_components/ng-file-upload/ng-file-upload.min.js',
           'bower_components/angular-local-storage/dist/angular-local-storage.js',
           'build/js/shared.module.js',
+          'build/js/walletDirectives.js',
```

Comment:
This is not good, because now you're making the landing page larger to load. As you suggested, it might be better to undo the `walletDirectives` module for now if you don't need it.